### ISSUE TITLE
ModuleNotFoundError: No module named 'kfp.deprecated'

### ISSUE THREAD

**Issue by drozzy** [kubeflow/pipelines#7478]
Pipelines Version: [1.8.1]
```
$ pip list | grep kfp 
kfp                      1.8.11
kfp-pipeline-spec        0.1.14
kfp-server-api           1.8.1
```

Following this tutorial:
https://www.kubeflow.org/docs/components/pipelines/tutorials/build-pipeline/#choose-and-compile-a-pipeline

After running this:
dsl-compile --py ${DIR}/sequential.py --output ${DIR}/sequential.tar.gz

I get this error:

```
Traceback (most recent call last):
  File "/Users/andriy/miniconda3/envs/mlpipeline/bin/dsl-compile", line 8, in <module>
    sys.exit(main())
  File "/Users/andriy/miniconda3/envs/mlpipeline/lib/python3.7/site-packages/kfp/compiler/main.py", line 138, in main
    mode,
  File "/Users/andriy/miniconda3/envs/mlpipeline/lib/python3.7/site-packages/kfp/compiler/main.py", line 108, in compile_pyfile
    __import__(os.path.splitext(filename)[0])
  File "pipelines/samples/core/sequential/sequential.py", line 17, in <module>
    from kfp.deprecated import dsl, compiler
ModuleNotFoundError: No module named 'kfp.deprecated'

```

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by zhuyaguang** [comment]
+1

**Comment by zhuyaguang** [comment]
i fix it by  replace "kfp.deprecated" to "kfp".

i think it caused by this PR 
https://github.com/kubeflow/pipelines/pull/7291

**Comment by ji-yaqi** [comment]
This is intended since our samples are now based off KFP v2 (main branch). `sdk/release-1.8` contains the v1 specific samples and naming. Please reopen if you have more issues with this.

**Comment by drozzy** [comment]
I don't understand what you mean. 
Shouldn't following the guide not generate errors? Because I did everything that was in the guide as far as I recall.

**Comment by drozzy** [comment]
It installed `Pipelines Version: [1.8.1]` following the guide. I.e. I didn't choose some specific version to install.

**Comment by ilaria-speranza** [comment]
I solved this issue by launching `git checkout 1.8.13` in the repo I cloned from Github
